README.md

```diff
@@ -71,6 +75,29 @@ distribution identifier in the database.
 * openwrt
 * docker
 
+### `rebuilder` (dictionary)
+
+Information about the rebuilder that provided this data.
+The allowed fields are described below.
+
+### `name` (string)
+
+The human-readable descriptive name of the rebuilder.
+
+#### Example property
+
+* `Debian official rebuilder`
+* `Foobar Corp. validation division`
+
+### `uri` (string)
+
+Indication to a human-readable website describing the rebuilder.
+
+#### Example property
+
+* `https://tests.reproducible-builds.org/debian/`
+* `https://validation.`oobar-corp.com/openwrt/`
```

Comment:
there is an backtick too much in the middle

Reply:
interesting, how did that happen…

thanks